<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />

    <param name="image_width" value="640" />
    <param name="image_height" value="480" />

    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="brightness" value="50"/>
    <param name="contrast" value="50"/>  
  
    <param name="framerate" value="5"/>

  </node>
  <node name="rqt_gui" pkg="rqt_gui" type="rqt_gui" />  
  
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/camera/image_color"/>
    <param name="autosize" value="true" />
  </node>


  <node name="tracker" pkg="mag_tracking" type="tracker_object_node" output="screen">
    	<remap from="/image" to="/usb_cam/image_raw"/>  
	<param name="scale" value="0.015"/>
  </node>
  
</launch>
